CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Changes by Version
 ==================
 
+3.6.1 (2017-11-21)
```

Comment:
new feature, need to bump minor: 3.7.0

Reply:
Whoops, that's what I meant to do initially.